public/docs/ts/latest/guide/browser-support.jade

```diff
@@ -0,0 +1,190 @@
+:marked
+  # Browser support #  
+  Angular supports most recent browsers. This includes the following specific versions:
+
+table
+  tr
+    th Chrome
+    th Firefox
+    th Edge
+    th IE
+    th Safari
+    th iOS
+    th Android
+    th IE mobile
+  tr
+    td latest
+    td latest
+    td 14
+    td 11
+    td 9
+    td 9
+    td Marshmallow (6.0)
+    td 11
+  tr
+    td
+    td
+    td 13
+    td 10
+    td 8
+    td 8
+    td Lollipop<br>(5.0, 5.1)
+    td
+  tr
+    td
+    td
+    td
+    td 9
+    td 7
+    td 7
+    td KitKat<br>(4.4)
+    td
+  tr
+    td
+    td
+    td
+    td
+    td
+    td
+    td Jelly Bean<br>(4.1, 4.2, 4.3)
+    td
+    
+    
+:marked
+  To ensure this compatibility, the unit tests of the framework are run on all of them for every single commit. 
+  This is done as part of the continuous integration setup of the project, using [SauceLabs](https://saucelabs.com/) and [Browserstack](https://www.browserstack.com).
+  
+  # Polyfills #
+  Angular is built on the latest standards of the web platform.
+  Targeting such a wide range of browsers can be challenging as they may not support all the functionalities of modern browsers.
+  
+  
+  To fill the gaps, the best solution is to use polyfills which "upgrade" old browsers by adding missing functionalities.
+  For each browser, some polyfill may be required to run any application, and some may be needed only when some features are used.
+
+.alert.is-helpful 
+  span As a developer, depending on the browsers you target and on the features you use, you'll need to load a set of polyfills.
+  span  The tables below will help you making the right choices.
+  
+.alert.is-important 
+  span The polyfills described here under are the ones which, others may be required for your application.
+  span  On top of that, they will not magically transform an old and slow browser into a modern and fast one.
+  
+:marked
+  ## Mandatory polyfills ##
+  For each supported browser, these are the polyfills which are required to run an application:
+  
+table
+  tr
+    th Browsers (desktop & mobile)
+    th Polyfills required
+  tr
+    td Chrome, Firefox, Edge, Safari 9+
+    td None
+  tr
+    td Safari 7 & 8, IE10 & 11, Android 4.1+
+    td ES6
+  tr
+    td IE9
+    td ES6<br>classList
+    
+    
+:marked
+  ## Optional polyfills ##
```

Comment:
"Optional browser features to polyfill" - isn't this more accurate? the list above doesn't list polyfills but feature that should be polyfilled.